CONTRIBUTING.md

```diff
@@ -231,7 +242,7 @@ Note that `.` from abbreviations, such as `i.e.`, count.
 The `$` character may be used in mixins to mark a semantic separation in the name, in other words it allows to separate
 the actual name of the variable and the namespace.
 
-`@Unique` fields must be prefixed with `quilt$`, but `@Unique` methods don't need prefixes.
+All mixin fields and methods (including those from duck interfaces) must be prefixed with `quilt$` in order to aid with debugging.
```

Comment:
While I understand that it's to help and make debugging better, I still think it shouldn't be the responsibility of mods but Mixin to prefix appropriately the injections.

Once Mixin is *finally* fixed (like damn it, it's been at least 4 years since the issue exist in the Fabric ecosystem) this change should be reverted.

Reply:
You have a good point; I'll have it reverted once Mixins are fixed